package.json

```diff
@@ -13,7 +13,8 @@
     "subgraph:codegen": "yarn workspace @project/subgraph codegen",
     "subgraph:build": "yarn workspace @project/subgraph build",
     "subgraph:deploy": "yarn workspace @project/subgraph deploy",
-    "react-app:build": "yarn workspace @project/react-app build",
+    "react-app:build-production": "PUBLIC_URL=https://stake.nucypher.network yarn workspace @project/react-app build",
```

Comment:
see this for explanation.
https://stackoverflow.com/questions/54340240/create-react-app-build-uncaught-syntaxerror-unexpected-token

With relative imports, anyone trying to directly load eg. https://stake.nucypher.network/new/worker/join,
statics would attempt to load from `https://stake.nucypher.network/new/worker/join/statics/js/something.js`
setting PUBLIC_URL makes them always load from root path.